src/add-puzzle.js

```diff
@@ -0,0 +1,44 @@
+import getConfig from './config.js';
+import { connect, WalletConnection, keyStores, utils } from 'near-api-js';
+import { generateNewPuzzleSeedPhrase } from './utils'
+import { parseSeedPhrase } from 'near-seed-phrase';
+
+export const addNewPuzzle = async (mungedLayout, layout, dimensions, prizeDeposit) => {
+    const nearConfig = getConfig(process.env.NEAR_ENV || 'testnet');
+    const keyStore = new keyStores.BrowserLocalStorageKeyStore();
+    nearConfig.keyStore = keyStore;
+    const near = await connect(nearConfig);
+    const wallet = new WalletConnection(near);
+
+    const cleanLayout = layout.map(clueAnswer => {
+        // remove answer and capitalize direction value
```

Comment:
would be nice to explain why we're capitalizing things here in this comment so future folks don't have to dig through the rust code to understand it

Reply:
done